src/main/flight/mixer.c

```diff
@@ -744,6 +744,14 @@ void mixTable(timeUs_t currentTimeUs, uint8_t vbatPidCompensation)
 
     // Calculate voltage compensation
     const float vbatCompensationFactor = vbatPidCompensation ? calculateVbatPidCompensation() : 1.0f;
+    
+    // Apply battery voltage compensation to throttle
+    if(currentPidProfile->vbatThrottleCompensationEnabled) {     
+        float slope = 200.0f / currentPidProfile->vbatThrottleCompensationSlope;
+        float vbatThrottleCompensationFactor = 
+            constrainf( (currentPidProfile->vbatThrottleCompensationVoltage - getBatteryAverageCellVoltage()) / slope, 0.5f, 1.5f );
```

Comment:
Physically correct compensation is V<sub>ref</sub>/V<sub>act</sub>

It's not far from line (http://www.wolframalpha.com/input/?i=3.9%2Fx+for+x+from+3.3+to+4.2), so linear approximation may work, but is a bit questionable.
Note that practical range ~ +=20%(over full V<sub>ref</sub> range), so compensation is relatively small

V<sub>ref</sub>/V<sub>act</sub> is approximated when slope = V<sub>ref</sub> (slope at V<sub>ref</sub> is correct).

This is a bit awkward to setup correctly using vbatThrottleCompensationSlope ... using scaling of V<sub>ref</sub>/V<sub>act</sub> may be better choice.

BTW: double division on slope is used unnecessarily ...